backend/layers/processing/process_validate.py

```diff
@@ -125,8 +125,8 @@ def validate_h5ad_file_and_add_labels(
         output_filename = CorporaConstants.LABELED_H5AD_ARTIFACT_FILENAME
         try:
             is_valid, errors, can_convert_to_seurat = self.schema_validator.validate_and_save_labels(
-                local_filename, output_filename
-            )
+                local_filename, output_filename, n_workers=1
```

Comment:
in our CLI code we should check if `dask.config` is already set and leave it alone if it is, then we don't need to pass n_workers to this function. That way we don't need to worry about keeping this value synced with the TF.